addons/live-featured-project/script.js

```diff
@@ -90,17 +93,13 @@ export default async function ({ addon, msg }) {
 
   // Start loading the players
 
-  if (forceAlternative && alternativePlayer !== "none") {
-    if (alternativePlayer === "turbowarp") loadTurboWarp();
-    else if (alternativePlayer === "forkphorus") loadForkphorus();
-  } else {
-    loadScratch();
-    iframeElement.addEventListener("load", () => {
-      if (iframeElement.contentDocument.querySelector(".not-available-outer") !== null) {
-        if (alternativePlayer === "turbowarp") loadTurboWarp();
-        else if (alternativePlayer === "forkphorus") loadForkphorus();
-        else stageElement.removeChild(wrapperElement);
-      }
-    });
-  }
+  if (player === "turbowarp") loadTurboWarp();
+  else if (player === "forkphorus") loadForkphorus();
+  else loadScratch();
+  iframeElement.addEventListener("load", () => {
+    if (iframeElement.contentDocument.querySelector(".not-available-outer") !== null) {
```

Comment:
I doubt this works with alternative viewers.

Reply:
![Screenshot_20230528_200117](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/adb424eb-95f4-4099-a63b-a80b84c174dc)

Reply:
I guess I'll make a call to Scratch's API then.